tests/conftest.py

```diff
@@ -10,7 +10,7 @@
 my_vcr = vcr.VCR(
     cassette_library_dir=join(dirname(realpath(__file__)), "cassettes"),
     path_transformer=vcr.VCR.ensure_suffix(".yaml"),
-    decode_compressed_response=True,
+    decode_compressed_response=False,
```

Comment:
If I understand this correctly:
- our cassettes contains decoded responses, because we saved them using `decode_compressed_response=True`.
- vcrpy v4.3.1 supports urlib3 v2 (good), but broke `decode_compressed_response` (bad) so we compensate by setting `decode_compressed_response` to the opposite of what we want.

Is this correct?

If so, it means we will need to revert this change when a new vcrpy version fixes that `decode_compressed_response` regression. I think it would be safer to pin vcrpy to `>=4.3.0,!=4.3.1,<4.4.0`, keep urlib3 pinned to `<2` and open an issue on vcrpy.